test/Rings-conformance-tests.jl

```diff
@@ -21,6 +21,104 @@ function equality(a::T, b::T) where T <: AbstractAlgebra.NCRingElement
    end
 end
 
+function test_mutating_op_like_zero(f::Function, f!::Function, A)
+   a = deepcopy(A)
+   a = f!(a)
+   @test equality(a, f(A))
+end
+
+function test_mutating_op_like_neg(f::Function, f!::Function, A)
+   for z in [zero(A), deepcopy(A)]
+      a = deepcopy(A)
+      z = f!(z, a)
+      @test equality(z, f(A))
+      @test a == A
+   end
+
+   a = deepcopy(A)
+   a = f!(a)
+   @test equality(a, f(A))
+end
+
+function test_mutating_op_like_add(f::Function, f!::Function, A, B)
+   for z in [zero(A), deepcopy(A), deepcopy(B)]
```

Comment:
Maybe add a comment why this loops is here? Same in other similar places (via copy&paste)

Reply:
I'll do that after lunch

Reply:
Done